k8s/karpenter/provisioners.yaml

```
@@ -17,11 +17,40 @@ spec:
       memory: 20Ti
 
   requirements:
-    # Only spin up arm64 nodes
+    # Only spin up arm64 and amd64 nodes
     - key: "kubernetes.io/arch"
       operator: In
       values: ["arm64", "amd64"]
 
+    # Don't provision nodes from certain instance families.
```

Sorry to be a stickler, but just so we have a record of why we're doing this
```suggestion
    # Don't provision nodes from certain instance families.
    # These are the instance families that don't support x86_v3
```